### PR TITLE
Always add an indexing docproc, also for streaming search.

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst PR

### CODE REVIEW

File: config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java [hunk 1]
```diff
@@ -1456,25 +1456,32 @@ private void assertIndexingDocprocEnabled(boolean indexed, boolean force, boolea
                 "</services>";
         VespaModel model = createEnd2EndOneNode(new TestProperties(), services);
         var searchCluster = model.getContentClusters().get("search").getSearch();
-        assertEquals(expEnabled, searchCluster.getIndexingDocproc().isPresent());
+        if (force) {
+            assertEquals("default", searchCluster.getIndexingDocproc().getClusterName("search"));
```
Comment (geirst) [inline config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java:1460]: Do we need to handle the tests on lines 1459-1466 differently? They all seem the same.

**Review by geirst — DISMISSED** [review]
👍

File: config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java (hunk 1 above)
Comment (baldersheim) [reply config-model/src/test/java/com/yahoo/vespa/model/content/ContentClusterTest.java]: Simplified

**Comment by jonmv** [comment]
This is fine, but it broke a test that relies on an additional field not being populated with streaming mode.

**Pull request merged**